# Author : Erik de Castro Lopo <erikd@mega-nerd.com>
# Description : Replace net version of DTD with local version.
# Debian Version : 0.16.1
# Date : Mon,  7 Sep 2009 16:14:58 +1000

Index: c2hs-0.16.0/doc/c2hs.xml
===================================================================
--- c2hs-0.16.0.orig/doc/c2hs.xml
+++ c2hs-0.16.0/doc/c2hs.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF8"?>
 <!DOCTYPE article PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
-  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd" [
+  "file:////usr/share/sgml/docbook/dtd/4.2/docbookx.dtd" [
     <!ENTITY C2hs 'C&#10132;Haskell'>
     <!ENTITY c2hs '<command>c2hs</command>'>
   ]>
